comit/Cargo.toml

```diff
@@ -41,12 +41,19 @@ tracing-futures = { version = "0.2", features = ["std-future", "futures-03"] }
 uuid = { version = "0.8", features = ["serde", "v4"] }
 
 [dev-dependencies]
+atty = "0.2"
```

Comment:
what does this library do?

Reply:
`let is_terminal = atty::is(Stream::Stdout);`